### ISSUE TITLE
HOLY BOOKS STUDIES

### ISSUE THREAD

**Issue by Loosifr** [decentraland/proposals#126]
My first idea for DCL VR was a spiritual library where any one could browse the various denomination  books of the world. 

Originally, I personally wanted to create a VR comic of my own translation of the Holy Bible, yet if there were a great library of holy books, I could largely expand this commission. There could be a study room where individuals can share their varying beliefs with one another and quietly debate the details of their own life, death and salvation. In this manner, an open interpretation of holy volumes can resurrect much mathematics and symbolism out of our current spiritual verbal graveyard.

 In short, I would like to see social interaction about racial and cultural differences protected in a peaceful manner. DCL will be a small Earth after all, and there has to be a place to commence healing!
